src/libraries/System.Net.Ping/tests/FunctionalTests/UnixPingUtilityTests.cs

```diff
@@ -32,11 +33,24 @@ public static void TimeoutIsRespected(int timeout)
             p.StartInfo.RedirectStandardError = true;
             p.StartInfo.RedirectStandardOutput = true;
 
+            bool destinationNetUnreachable = false;
+            p.OutputDataReceived += delegate (object sendingProcess, DataReceivedEventArgs outputLine)
+            {
+                if (outputLine.Data?.Contains("Destination Net Unreachable", StringComparison.OrdinalIgnoreCase) == true)
```

Comment:
Does this text get localized on any OSes?

Reply:
On Windows it is, but this file is specific to Unix where I haven't seen this output being localized.

We also already rely on the English strings in `ParseReturnedPacketSize` and `ParseNumPingsSent`.

Reply:
```suggestion
                if (outputLine.Data?.Contains("Destination Net Unreachable", StringComparison.OrdinalIgnoreCase))
```
maybe we should have an analyzer for `== true` and `!= false`

Reply:
@danmoseley this doesn't work because of the null conditional operator so we need the explicit comparison.